### ISSUE TITLE
NullInjectorError DatePipe after #160 (v4.1.0)

### ISSUE THREAD

**Issue by infnada** [dbfannin/ngx-logger#163]
After updating ngx-logger to 4.1.0 #160 i get this error.

ERROR NullInjectorError: StaticInjectorError(AppModule)[CustomNGXLoggerService -> DatePipe]: 
  StaticInjectorError(Platform: core)[CustomNGXLoggerService -> DatePipe]: 
    NullInjectorError: No provider for DatePipe!

I'm using ngx-logger as a library like: https://github.com/infnada/SysOS/blob/anyOpsOS/projects/libraries/logger/src/lib/services/anyopsos-lib-logger.service.ts

**Comment by MicMicMon** [comment]
Oops, indeed, you need to have DatePipe imported in your `providers`. Quickfix for now:

Add to your providers in main module `app.module.ts`:
```
import { DatePipe } from '@angular/common';
// ...
providers: [ ..., DatePipe]
```

I guess `DatePipe` now has to be imported by default by the package?

**Comment by dbfannin** [comment]
This should now be fixed in ngx-logger@4.1.1. Thanks @MicMicMon for the quick fix.

**Comment by MicMicMon** [comment]
Sorry for the trouble!
